duckduckgo/iOS
# favourites by default

Task/Issue URL: https://app.asana.com/0/392891325557410/1127121491900615
Tech Design URL:
CC:

**Description**:

An experiment to test favorites and center search by default.

When adding a favorite from the home screen it prompts for a URL only, then takes the host name with `www.` stripped as the title.

Variants:

* `ma`: current production
* `mb`: favorites and center search by default

Pixels:

    case homeScreenShown = "mh"
    case homeScreenSearchTapped = "mh_st"
    case homeScreenFavouriteLaunched = "mh_fl"
    case homeScreenAddFavorite = "mh_af"
    case homeScreenAddFavoriteOK = "mh_af_o"
    case homeScreenAddFavoriteCancel = "mh_af_c"
    case homeScreenEditFavorite = "mh_ef"
    case homeScreenDeleteFavorite = "mh_df"

Note: currently in design review

**Steps to test this PR**:
1.  Fresh install, ensure that both variants manifest.
1. Ensure pixels are fired at appropriate times.
1. Test migration
1. General testing of favorites functionality.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
